config/manager/manager.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: operator
+  labels:
+    operator: topolvm
+spec:
+  selector:
+    matchLabels:
+      app: topolvm-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: topolvm-operator
+    spec:
+      serviceAccountName: topolvm-operator
+      containers:
+        - name: topolvm-operator
+          # set the stable version
+          image: controller:latest
+          command:
+            - /topolvm
+          args:
+            - operator
+          env:
+            - name: TOPOLVM_LOG_LEVEL
+              value: "DEBUG"
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: IS_OPERATOR_HUB
+              value: "0"
```

Comment:
IS_OPERATOR_HUB shoudl set "1"

because olm could not create csidriver and psp resource, so we should turn on the swich make operator create csidriver and psp.